device_operation/include/device_reduce_blockwise.hpp

```diff
@@ -0,0 +1,350 @@
+#ifndef DEVICE_REDUCE_BLOCKWISE_HPP
+#define DEVICE_REDUCE_BLOCKWISE_HPP
+
+#include <iostream>
+#include <sstream>
+#include "device.hpp"
+#include "device_reduce.hpp"
+#include "device_reduce_common.hpp"
+#include "gridwise_2d_reduction_blockwise.hpp"
+
+namespace ck {
+namespace tensor_operation {
+namespace device {
+
+template <typename InDataType,
+          typename AccDataType,
+          typename OutDataType,
+          int Rank,
+          typename InnerDims,
```

Comment:
ReduceDims

Reply:
I saw  cutlass using  `OuterDims` for invariant dimensions,  `InnerDims` for to be reduced dimensions.

Reply:
Please be as explicit as possible when naming. We are not following cutlass naming

Reply:
Will change to  `InvariantDims` and `ReduceDims`